Set groupOwner in CSW harvester - fix #2188

fgravin opened the pull request:
Set groupOwner to all harvested metadata as required from harvester admin UI.

fgravin commented:
fix #2188

fgravin commented:
@josegar74 I have set the group owner in all harvesters.
Wonder if you should set it on metadata update as it's done on metadata insert ? I would say yes.

josegar74 commented:
I guess so, but with actual code the update is done only if changes are detected in the metadata change date. Suppose this case:

1) User creates a harvester and assigns group/user owner.

2) Harvest the metadata and gets the proper values.

3) Edits the harvester and changes group or user, but the metadata is not updated due to no date changes, so at least with current code will not be changed the group or user if updating the update code to update group/user.

I see 2 options:

- Don't allow change group/user on harvester edit, just on creation (maybe too restrictive)

- Update harvester update logic, so the metadata is not updated if no changes detected, but the code to update group/user is applied in any case.

Also a small change, please check to update the following help text.I would change it to: 

`Group which owns the harvested records. Only the catalog administrator or user admin of this group can manage this harvester node.`

![group-harvester-help](https://user-images.githubusercontent.com/1695003/30852048-ca2217de-a2aa-11e7-9002-c6de6a474135.png)

fgravin commented:
> Don't allow change group/user on harvester edit, just on creation (maybe too restrictive)

I keep this i can't handle huge changes right now.
I've added the translation and changed the one in french in transifex, ger an ita are not translated at all yet.

fgravin commented:
Travis VM crashes somtimes .. not related with PR changes :/
